chart/tests/test_pgbouncer.py

```diff
@@ -427,7 +455,7 @@ def test_default_exporter_secret(self):
     def test_exporter_secret_with_overrides(self):
         connection = self._get_connection(
             {
-                "pgbouncer": {"enabled": True},
+                "pgbouncer": {"enabled": True, "metricsExporterSidecar": {"sslmode": "disable"}},
```

Comment:
```suggestion
                "pgbouncer": {"enabled": True, "metricsExporterSidecar": {"sslmode": "require"}},
```

The intent of this test is to ensure non-defaults can be set.